Resources/Server Info/Guidebook/Law/Charges.xml

```diff
@@ -0,0 +1,118 @@
+﻿<Document>
+# Charges & Sentencing
+
+This section lists all [color=#a4885c]criminal charges[/color], their [color=#a4885c]sentences[/color] as well as how to sentence a crime.
+
+## Sentencing a Crime
+- Choose one of the charges you are convicting someone of as a [color=#a4885c]Primary Charge[/color]. Add brig time between its minimum and maximum.
```

Comment:
Space Law says the charge with highest minimum is the primary.

Reply:
This is true, and will fix. The intent was to let the sentencing officer pick to be as lenient or strict as they deem fit, but I forgot that I also wanted minimums to be always observed, hence your correct comment. But since a higher minimum also corresponds to higher maximums (I just checked), this is fulfilled.

TL;DR: Will fix, thank you. Hooliganism is gonna have "Brig until compliant" as Compound Limit as well.

Reply:
Fixed with 1ea5d73.